insights/specs/default.py

```diff
@@ -837,6 +837,8 @@ def rhev_data_center(broker):
     rpm_V_packages = simple_command("/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo", keep_rc=True)
     rsyslog_conf = simple_file("/etc/rsyslog.conf")
     samba = simple_file("/etc/samba/smb.conf")
+    # TODO also parse the stderr output of testparm -s
+    samba_smb_conf = simple_command("/usr/bin/testparm -s /etc/samba/smb.conf 2>/dev/null")
```

Comment:
@cryptomilk this command will not work with the  `2> /dev/null` because all commands are run without the shell.

Also since the client adds this information to the archive as a file you will need to include a `simple_file` spec in `insights/specs/insights_archive.py`.  Use the `mangle` command in insights-core to determine what the file path  should be.  For example:
```
$ mangle "/usr/bin/testparm -s /etc/samba/smb.conf"
```

Reply:
Done. Thanks!